english/relation_prediction.md

```diff
@@ -44,7 +44,8 @@ As a way to overcome this problem, [Dettmers et al. (2018)](https://arxiv.org/ab
 The test set is composed of triplets, each used to create two test instances, one for each entity to be predicted. Since each instance is associated with a single true entity, the maximum value for all metrics is 1.00.
    
 | Model           | H@10 | H@1 | MRR | Paper / Source | Code | 
-| ------------- | :-----:| :-----:| :-----:| --- | --- | 
+| ------------- | :-----:| :-----:| :-----:| --- | --- |
+| CoKE (Wang, Quan, et al, 2019) | 60.3 | 47.2 | 51.5 | [CoKE: Contextualized Knowledge Graph Embedding](https://arxiv.org/abs/1911.02168) | [Official](https://github.com/PaddlePaddle/Research/tree/master/KG/CoKE) |
```

Comment:
I'm a little confused. Can you help me? 

I see that the H@10 in the paper is 0.553 for WN18RR. While the above entry says 60.3. 

Maybe I'm looking at the wrong dataset or wrong place in the paper. Can you please point me in the right direction? 

![image](https://user-images.githubusercontent.com/3250749/84585968-aea70280-ae32-11ea-87f6-72f218a313eb.png)

Reply:
Sorry NirantK, It was my fault. The metrics results of our recent work has not been updated yet. I will close this PR, and re-submit with the metrics consistent with linked paper.